test-suite-kotlin/src/test/kotlin/io/micronaut/kafka/docs/consumer/reactive/ProductListener.kt

```diff
@@ -0,0 +1,34 @@
+package io.micronaut.kafka.docs.consumer.reactive
+
+import io.micronaut.configuration.kafka.annotation.KafkaKey
+import io.micronaut.configuration.kafka.annotation.KafkaListener
+import io.micronaut.configuration.kafka.annotation.Topic
+import io.micronaut.context.annotation.Requires
+import io.micronaut.core.annotation.Blocking
+import io.micronaut.kafka.docs.Product
+import reactor.core.publisher.Mono
+
+@Requires(property = "spec.name", value = "ReactiveProductListenerTest")
+@KafkaListener
+class ProductListener {
+
+    // tag::method[]
+    @Topic("reactive-products")
+    fun receive(@KafkaKey brand: String,  // <1>
+        productFlowable: Mono<Product>): Mono<Product> { // <2>
+        return productFlowable.doOnSuccess { (name): Product ->
+            println("Got Product - $name by $brand") // <3>
```

Comment:
Us `LOG` not `println`

Reply:
changed it, here: https://github.com/micronaut-projects/micronaut-kafka/pull/832/commits/bfa2e2caf5233ff5064c5508d8b2eb2e08726b37